RuleDocumentation/UseIdenticalMandatoryParametersForDSC.md

```diff
@@ -4,13 +4,24 @@
 
 ## Description
 
-The `Get-TargetResource`, `Test-TargetResource` and `Set-TargetResource` functions of DSC Resource must have the same mandatory parameters.
+For script based DSC resources the `Get-TargetResource`, `Test-TargetResource` and `Set-TargetResource` functions must have identical mandatory parameters that are also the keys in the corresponding `mof` file.
```

Comment:
But isn't the rule supposed to look for identical mandatory parameters in `Get/Test/Set` functions only if they are present as keys in the mof file?